Tasks/AzureIoTEdge/package-lock.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "azure-iot-edge-vsts-extension-task",
```

Comment:
Please ensure that the OSS dependencies and the specified versions are approved in OSS tool and have been code scanned. You might also need a ThirdPartyNotices.txt. @vincentdass as FYI.

Reply:
Added TPN